artifactory/utils/commandsummary/utils.go

```diff
@@ -67,6 +67,10 @@ func addGitHubTrackingToUrl(urlStr string, section summarySection) (string, erro
 
 	// Get the query parameters and add the GitHub tracking parameters
 	queryParams := parsedUrl.Query()
+	// Tracking source
+	queryParams.Set("s", "1")
```

Comment:
I suggest adding some description regarding these values, and perhaps making them constants.